posts/navarroaxel/11.npm-query-for-installed-packages.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Querying installed packages with npm query
+published: true
+description: Check your installed dependencies using CSS selectors format.
+tags: 'javascript, webdev, node, react'
+cover_image: ./assets/npm-query-cover.jpg
+id: 1209907
+---
+
+Do you know if you use npm dependencies with `postinstall` scripts? Do you have a package installed twice? Since version 8.18.0 we can use [`npm query`](https://docs.npmjs.com/cli/v8/commands/npm-query) to find packages in the `node_modules` directory matching a specific query.
```

Comment:
```suggestion
Do you know if you have any package that makes use of postinstall scripts? Do you have a package installed twice? Since version 8.18.0 we can use [`npm query`](https://docs.npmjs.com/cli/v8/commands/npm-query) to find packages in the `node_modules` directory matching a specific query.
```